charts/microcks-operator/templates/_helpers.tpl

```diff
@@ -0,0 +1,24 @@
+{{- define "microcks-operator.name" -}}
+{{- default .Chart.Name .Values.nameOverride | trunc 63 | trimSuffix "-" -}}
+{{- end -}}
+
+{{- define "microcks-operator.fullname" -}}
+{{- if .Values.fullnameOverride -}}
+{{- .Values.fullnameOverride | trunc 63 | trimSuffix "-" -}}
+{{- else -}}
+{{- $name := default .Chart.Name .Values.nameOverride -}}
+{{- if contains $name .Release.Name -}}
+{{- .Release.Name | trunc 63 | trimSuffix "-" -}}
+{{- else -}}
+{{- printf "%s-%s" .Release.Name $name | trunc 63 | trimSuffix "-" -}}
+{{- end -}}
+{{- end -}}
+{{- end -}}
+
+{{- define "microcks-operator.labels" -}}
+app.kubernetes.io/name: {{ include "microcks-operator.name" . }}
+app.kubernetes.io/instance: {{ .Release.Name }}
+app.kubernetes.io/managed-by: {{ .Release.Service }}
+app.kubernetes.io/part-of: microcks
```

Comment:
`app.kubernetes.io/part-of` is already used on managed resources with the value being the name of the `MicrocksInstall` CR - most of the time being `microcks`. Don't you think it wouldn't be better to put `microcks-operator` here?

Reply:
Or - best - to reuse `{{ include "microcks-operator.name" . }}`?